File: Inversion/parallel/NA/src/hv_subs/PROGRAMS.330/CALPLOT/testf/width.f
	call pinitf('WIDTH.PLT')
	call gunit('in')
	do 1000 ifont=1,4
		call gfont(ifont)
		do 100 i = 16,111
			ix = mod((i-1),16)
			iy = (i-1)/16
			xx = 1.0 + ix*0.5
			yy = 7.0 - (iy-1)*1.0
			ang = 0.0
			call gwidth(0.05)
			call symbol(xx,yy,0.5,char(i),ang,-1)
			call gwidth(0.0)
  100		continue
		call frame()
 1000	continue
	call pend()
	end

File: Inversion/parallel/NA/src/hv_subs/PROGRAMS.330/CALPLOT/testc/width.c
#include "calplot.h"
main()
{
	int ifont, i, ix, iy;
	float xx, yy, ang;
	float zero = 0.0;
	float height = 0.5;
	float width = 0.05;
	char str[2];
	int mone = -1;
	str[1] = '\0';
	pinitf("WIDTH.PLT");
	gunit("in");
	for(ifont=1;ifont<=4;ifont++){
		gfont(ifont);
		for(i=16;i<=111;i++){
			ix = (i-1)%16;
			iy = (i-1)/16;
			xx = 1.0 + ix*0.5;
			yy = 7.0 - (iy-1)*1.0;
			ang = 0.0;
			gwidth(width);
			str[0] = i;
			str[1] = '\0';
			symbol(xx,yy,height,str,ang,mone);
			gwidth(zero);
		}
		frame();
	}
	pend();
}
